### PR TITLE
Superadmins

### PR DESCRIPTION
**Pull request by jjlgao**
Currently everything working except I haven't figured out how to make the app yell "are you sure?" at you yet.

### CODE REVIEW

File: app/views/admins/super.html.erb [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+                        </tr>
+                            <% Admin.all.each do |a| %>
+                                <% if a.id != @admin.id %>
+                                    <tr>
+                                        <td><%= a.name %></td>
+                                        <td>
+                                            <%= radio_button_tag "transfer_admin", a.id %>
+                                        </td>
+                                        <td>
+                                            <%= link_to "Delete #{a.name}", admin_path(:id => a.id), class: "btn", data: {method: "delete", confirm: "Are you sure you want to delete #{a.name}?"} %>
```
Comment (dasolyoon) [inline app/views/admins/super.html.erb]: I think MDS wanted checkboxes so he can delete multiple people at once but I guess it's fine

Comment (dasolyoon) [inline app/views/admins/super.html.erb]: Also after the delete action probably should redirect to the edit admins page again instead of to the home page

Comment (jjlgao) [inline app/views/admins/super.html.erb]: It's impossible to do both checkboxes and radio buttons without having them all have one submit button (e.g. every time MDS wants to delete someone he has to make sure he's not trying to transfer superadmin powers too). Is that fine?

Comment (jjlgao) [inline app/views/admins/super.html.erb]: Or separating out transferring and deleting into two different tables, which is what I'm implementing now.

**Pull request merged**